MantidPlot/src/Mantid/MantidMatrixModel.cpp

```diff
@@ -44,8 +44,7 @@ void MantidMatrixModel::setup(const Mantid::API::MatrixWorkspace *ws, int rows,
   m_colNumCorr = 1;
   m_endRow = m_rows - 1;
   m_startRow = start >= 0 ? start : 0;
-  m_mon_color =
-      QApplication::palette().color(QPalette::Active, QPalette::ToolTipBase);
+  m_mon_color = QColor(255, 253, 209);
```

Comment:
Can't we do better than hardcoded numbers?

Reply:
I don't think there's a simple way to pass the original colour from [here](https://github.com/mantidproject/mantid/blob/master/MantidPlot/src/ApplicationWindow.cpp#L11561-L11585) to where we are now.

Reply:
I would get the color of whatever is the background and change the color of the text in the cell to have ```
text_hue=(bg_hue+180)%360;
text_saturation=(bg_saturation+128)%256;
text_value=(bg_value+128)%256;
```
This idea is from https://doc.qt.io/archives/qq/qq26-adaptivecoloring.html